app/src/main/res/xml/settings.xml

```diff
@@ -25,6 +25,14 @@
         android:layout="@layout/bg_settings_section_security">
         <com.seafile.seadroid2.ui.SwitchPreferenceCompat
             android:defaultValue="false"
+            android:key="client_encrypt_switch_key"
```

Comment:
According to our product manager, it\`s better to keep the same behaviour after version upgrade. Users may not noticed the new feature and they may not want to be forced to turn on the feature as well.